frontend/src/lib/constants.ts

```diff
@@ -73,6 +73,11 @@ export const QUERY_REFETCH_OFF_OPTIONS = {
   refetchIntervalInBackground: false,
 };
 
+export const QUERY_REFETCH_LIMITED_OPTIONS = {
```

Comment:
if we are going to apply the refetch of all queries on window focus , it is better to provide the parameter and the config to the root `QueryClient` that gets pass to the provider rather than passing it as an option to all the queries.

FYI @Haarolean

Reply:
Currently it is not being passed to all queries just ones that are editable so that the fetching does not overwrite any temporary changes made. Should this be provided to all queries instead?

Reply:
Nope, as we discussed, only for the views which support editable parts: [source](https://github.com/kafbat/kafka-ui/issues/348#issuecomment-2094976884)

cc @Mgrdich